networks.js

```diff
@@ -101,7 +101,7 @@ const networks = {
     accounts,
     verifyApiKey: process.env.POLYGONSCAN_API_KEY || "UNSET",
     chainId: 80001,
-    confirmations: DEFAULT_VERIFICATION_BLOCK_CONFIRMATIONS,
+    confirmations: DEFAULT_VERIFICATION_BLOCK_CONFIRMATIONS * 4,
```

Comment:
That looks a little weird. Does it actually hurt to keep it at DEFAULT?

Reply:
do we want to have network specific defaults?

Reply:
I'll just set this to 10 blocks. I've noticed that it can be a problem if we don't wait long enough before verifying contracts.

Reply:
10 blocks on Sepolia is 2 minutes, way too slow IMO